src/motor-operations.h

```diff
@@ -45,6 +46,17 @@ struct LinearSegmentSteps {
   int steps[BEAGLEG_NUM_MOTORS]; // Steps for axis. Negative for reverse.
 };
 
+// Struct used to return data about the currently executed steps
+// and the status of the auxes.
+struct RealtimeStatus {
```

Comment:
RealtimeStatus is not really a good name. It is something we get back when asking with `GetRealtimeStatus()`, but what we get back is fundamentally only really a current configuration of where the motors are.

We should probably have a struct not called RealtimeStatus, but PhysicalPosition or PhysicalStatus or something (naming not quite clear yet).
So say
```
struct PhysicalPosition {
    int steps[BEAGLEG_NUM_MOTORS];
    unsingned short aux_bits;
};
```
(the fact that there are aux bits makes 'position' a bit bad, maybe it should be called `PhysicalStatus`).

Then we can use this type in `LinearSegmentSteps`

```
struct LinearSegmentSteps {
    float v0;
    float v1;
    PhysicalPosition target;
}
```

.. and when asking the realtime position.
```
 virtual void GetRealtimeStatus(PhysicalPosition *status);
```

I think it helps in reducing the number of different types used, and it makes it more clear what the intend is.